scanner/src/test/resources/illumina/130417_M00146_0021_000000000-A2WDF/reference.json

```diff
@@ -1,29 +1,28 @@
 {
-  "bclCount": 28,
-  "callCycle": 242,
-  "chemistry": "V2",
-  "completionDate": "2013-04-18T14:36:31Z",
-  "containerModel": "15028382",
-  "containerSerialNumber": "000000000-A2WDF",
-  "healthType": "COMPLETED",
-  "imgCycle": 242,
-  "indexLengths": [],
-  "laneCount": 1,
-  "numCycles": 242,
-  "numReads": 2,
-  "pairedEndRun": true,
-  "platform": "Illumina",
-  "poolNames": {
-    "1": "Unique_Sample_ID"
-  },
```

Comment:
Why was this removed?

Reply:
Ah, we actually stopped the Illumina processor from setting these a while back because the names it was using were not correct. The tests never actually checked this value anyway though